.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:chakracore-10.13.0
```

Comment:
any particular reason for chakracore? Chakracore is the JavaScript engine in microsoft edge. Node.js can target it instead of Chrome v8 but this is a really unusual choice unless you're doing something really Microsoft specific

Reply:
no, not at all. I was only looking at the version of node - didn't think about the codename at all. I thought it's just the codename of the nodejs release. Will fix it.